_includes/giscus.html

```
@@ -0,0 +1,18 @@
+{% if page.giscus %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)